pyanaconda/payload/nullpayload.py

```diff
@@ -0,0 +1,38 @@
+# nullpayload.py
+#
+# Copyright (C) 2018  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+from pyanaconda.core import util
+from pyanaconda.payload import Payload, PayloadError, versionCmp
+
+class NullPayload(Payload):
+    """ A NullPayload assumes the target alrady contains a system and does nothing."""
+
+    @property
+    def spaceRequired(self):
+        return 0
+
+    @property
+    def kernelVersionList(self):
```

Comment:
I think this should borrow the implementation from livepayload _upateKernelVersionList from the /boot/ contents. This would possibly allow bootloader installation to work in some cases (eg. UEFI).